rdr_service/resource/schemas/genomics.py

```diff
@@ -221,21 +221,36 @@ class GenomicGCValidationMetricsSchema(Schema):
     site_id = fields.String(validate=validate.Length(max=80))
     chipwellbarcode = fields.String(validate=validate.Length(max=80))
     idat_green_received = fields.Int16()
+    idat_green_deleted = fields.Int16()
```

Comment:
Same as above for these fields. Can we move the new `*_deleted` fields to the end of the definition?

Reply:
Done